windows/src/Makefile

```diff
@@ -6,24 +6,24 @@
 
 default:
     @echo The following make targets are available:
-    @echo ����build�����������������builds everything except help files (-DBUILDHELP to build help)
-    @echo ����install ���� ���� ����installs packages into Delphi IDE (run after a build)
-    @echo ����clean�������������� ��removes all temp files
-    @echo ����release���������������make a release
-    @echo ����web-install�����������sends the documentation to the website
-    @echo ����test-certificate������create a test certificate + CA (and install test CA)
-    @echo ����signcode��������������sign the build artifacts
+    @echo ����build�����������������builds everything except help files (-DBUILDHELP to build help)
+    @echo ����install ���� ���� ����installs packages into Delphi IDE (run after a build)
+    @echo ����clean�������������� ��removes all temp files
+    @echo ����release���������������make a release
+    @echo ����web-install�����������sends the documentation to the website
+    @echo ����test-certificate������create a test certificate + CA (and install test CA)
+    @echo ����signcode��������������sign the build artifacts
     @echo.
-    @echo ����Build flags:
-    @echo ����������DEBUG ����������Include debug information
-    @echo ����������SC_TIMESTAMP ���Don't timestamp certificates (for internal releases only, offline)
-    @echo ����������BUILDHELP ������Build the helpfiles (implicit when 'make release')
-    @echo ����������LINT ���� ���� �Turns on $MESSAGE hints and warnings and all extra string warnings
-    @echo ����������QUIET ���� ���� Ignore hints and warnings
-    @echo ����������RELEASE_OEM ����Build executables for third party products in oem folder (e.g. FirstVoices)
+    @echo ����Build flags:
+    @echo ����������DEBUG ����������Include debug information
+    @echo ����������SC_TIMESTAMP ���Don't timestamp certificates (for internal releases only, offline)
+    @echo ����������BUILDHELP ������Build the helpfiles (implicit when 'make release')
+    @echo ����������LINT ���� ���� �Turns on $MESSAGE hints and warnings and all extra string warnings
+    @echo ����������QUIET ���� ���� Ignore hints and warnings
+    @echo ����������RELEASE_OEM ����Build executables for third party products in oem folder (e.g. FirstVoices)
     @echo.
-    @echo ����To view help on the build process, in /src/inst, run:
-    @echo ����������make help
+    @echo ����To view help on the build process, in /src/inst, run:
+    @echo ����������make help
```

Comment:
I'm assuming these characters are harmless

Reply:
Ugh, character encoding strikes again! Will fix (was a char 160 in cp1252 or possibly even DOS codepage because Borland Make is old)

Reply:
'tis cp1252.